src/components/Personalization/handlers/createProcessHtmlContent.js

```diff
@@ -40,9 +41,18 @@ export default ({
       storeInteractionMeta,
     );
 
+    const renderStatusHandler = createRenderStatusHandler(
+      item.getProposition().getScopeType(),
+      item.getId(),
+    );
+
     return {
       render: () => {
-        return modules[type](item.getData(), decorateProposition);
+        return modules[type](
+          item.getData(),
+          decorateProposition,
+          renderStatusHandler,
```

Comment:
Do you need to add the renderStatusHandler here too?

https://github.com/adobe/alloy/blob/f7b6e1024ccb69bd87c0cd881ac7f20c00ac4fa4/src/components/Personalization/handlers/createProcessInAppMessage.js#L81